README.md

```diff
@@ -19,8 +19,9 @@ Using
 5. Create a token for a specific user and for the service 'User key authentication web service' (Admin > Plugins > Web services > Manage tokens)
 6. Make sure that the "web service" user has 'auth/userkey:generatekey' capability.
 7. Authorise the "web service" user: Admin > Plugins > Web services > External services, select 'Authorised users' for the web service, and add the user.
-8. Configure your external application to make a web call to get login URL.
-9. Redirect your users to this URL to be logged in to Moodle.
+8. Enable the userkey authentication plugin issue #93, more info https://github.com/catalyst/moodle-auth_userkey/issues/93#issuecomment-1669268077
```

Comment:
Thanks @baffourt for the proposed change. 

Actually if you look at step 2, it's already saying "Enable and configure just installed plugin". So maybe split this step and add more details? 

like  
`
2. Enable the userkey authentication plugin (Site administration -> Plugins -> Authentication and then enabled Userkey).
3. Configure the plugin. Set required Mapping field, User key life time, IP restriction and Logout redirect URL.` 

Also No need having links to the issue in the text. Thanks!